File: src/main/java/ru/ssau/tourism/entities/Tourist.java
package ru.ssau.tourism.entities;

import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Setter
@NoArgsConstructor
@EqualsAndHashCode(of = {"id"})
public class Tourist {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    @Column
    private String name;
    @Column
    private String surname;
    @Column
    private String fatherName;

    public Long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getSurname() {
        return surname;
    }

    public String getFatherName() {
        return fatherName;
    }
}

File: src/main/kotlin/ru/ssau/tourism/entities/Tourist.kt
package ru.ssau.tourism.entities

import javax.persistence.*

@Entity
class Tourist {
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	var id: Long? = null
	@Column
	var name: String? = null
	@Column
	var surname: String? = null
	@Column
	var fatherName: String? = null
}